torch/csrc/jit/passes/peephole.cpp

```diff
@@ -25,7 +25,7 @@ struct PeepholeOptimizeImpl {
       : aliasDb_(nullptr),
         graph_(graph),
         changed_(true),
-        addmm_fusion_enabled_(addmm_fusion_enabled) {
+        onnx_export_(addmm_fusion_enabled) {
```

Comment:
If the flag is indeed only used by onnx, please update all `addmm_fusion_enabled` to `onnx_export` to better reflect the semantics.